portfolio/src/main/webapp/script.js

```diff
@@ -39,5 +39,9 @@ function addRandomFact() {
 async function getGreetingUsingAsyncAwait() { 
   const response = await fetch('/data');
   const greeting = await response.text();
-  document.getElementById('greeting-container').innerHTML = greeting;
+  const greetingContainer = document.getElementById('greeting-container');
+  greetingContainer.innerText = greeting;
+  greetingContainer.innerText.style.color = "#FEE9E1";
+  greetingContainer.innerText.style.font = "12px georgia";
+  greetingContainer.innerText.padding = "10px";
```

Comment:
I feel like this might be better for a CSS file - setting CSS in JS is hard to maintain and generally not the way to go.

I won't block this PR for it, since there are examples of it already submitted (more bad starter code?), but I will enforce that for the capstone.

Reply:
Noted for the future thank you!